portal/src/main/webapp/WEB-INF/jsp/dashboard/dashboard.jsp

```diff
@@ -326,40 +352,33 @@
                                 if (typeof response === 'string') {
                                     response = JSON.parse(response); 
                                 }
-                                $("#show_study_details").css('display','block');
-                                $("#study_name").html(response['data']['name']);
-                                $("#study_desc").html(response['data']['description']);
-                                //$("#submit_button").css('display','none');
+                                var _description = response['data']['description'] || '';
```

Comment:
Over here we invoke session service to get name and description of virtual study. But in this case we can directly set name to `Selected Study` or `Selected Studies` and description directly if its a virtual study

Reply:
Ture, but thinking about that the user can still modify the virtual study name and description in the future, we may want to keep this.

Reply:
👍